dbbackup/tests/commands/test_dbbackup.py

```diff
@@ -8,7 +8,7 @@
 from django.utils import six
 
 from dbbackup.management.commands.dbbackup import Command as DbbackupCommand
-from dbbackup.dbcommands import DBCommands, MongoDBCommands
+from dbbackup.db import get_connector
```

Comment:
I'm a bit wondering if it's a good idea with this relocation of objects by importing them into `dbbackup.db.__init__` -- I think it's better to not do that and just have them live in their respective modules. It might strategically serve to avoid circularity in imports, although they're not possible at present.